icu4c/source/data/brkitr/ja.txt

```diff
@@ -238,4 +238,782 @@ ja{
         "ーリンズ",
         "ーン",
     }
+    model {
+        "BB2:108120:1800",
+        "UB3:107:271",
+        "UB3:120:-857",
```

Comment:
It looks like you are splitting each of these strings at the last `:`, into a key string before and an integer after. In order not to have to do that, and not have to parse the last part into an integer, I suggest you store separate string and integer values:

alternating
```
    model {
        "BB2:108120", :int{1800},
        "UB3:107", :int{271},
        "UB3:120", :int{-857},
    ...
```
or parallel arrays
```
    modelKeys {
        "BB2:108120",
        "UB3:107",
        "UB3:120",
    ...
    }
    modelValues:intvector {
        1800,
        271,
        -857,
    ...
```

Reply:
Done.